spec/FlexibleHttpClientSpec.php

```diff
@@ -92,3 +88,22 @@ function it_does_not_emulate_a_client(FlexibleHttpClient $client, RequestInterfa
         $this->sendAsyncRequest($asyncRequest);
     }
 }
+
+class CombinedHttpClient implements HttpClient, HttpAsyncClient
+{
+    /**
+     * @param RequestInterface $request
+     */
+    public function sendRequest(RequestInterface $request)
+    {
+        // TODO: Implement sendRequest() method.
```

Comment:
then lets call it StubCombinedClient and not write TODO here but just have empty body. and a class phpdoc that says this is a stub for testing.